sermant-backend/src/main/java/io/sermant/backend/dao/redis/RedisClientImpl.java

```diff
@@ -105,7 +105,7 @@ public boolean addEvent(Event event) {
 
     @Override
     public boolean addInstanceMeta(InstanceMeta instanceMeta) {
-        redisOperation.setex(instanceMeta.getMetaHash(), backendConfig.getHeartbeatEffectiveTime(),
+        redisOperation.setex(instanceMeta.getMetaHash(), backendConfig.getHeartbeatEffectiveTime() / 1000,
```

Comment:
You are advised to add the psetex method to the RedisOperation interface to set the expiration time in milliseconds. Use the jedis.psetex or jedisCluster.psetex method to implement the method.

Reply:
I add psetex method in RedisOperation, PTAL